framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -636,6 +636,16 @@ private AnnotationMirror createBooleanAnnotationMirror(List<Boolean> values) {
     }
 
     @Override
+    public List<AnnotationMirror> getJaifInferenceIgnoredAnnotations() {
+        List<AnnotationMirror> out = super.getJaifInferenceIgnoredAnnotations();
```

Comment:
Removed this method. See https://github.com/typetools/checker-framework/pull/477#discussion_r45111547